ci/lint/format.sh

```diff
@@ -138,13 +138,13 @@ MYPY_FLAGS=(
 )
 
 MYPY_FILES=(
-    # Relative to python/ray
```

Comment:
This was necessary to change, or else the new `mypy` version complains that there cannot be a `types.py` file in the top level directory:
```
Running mypy on autoscaler/node_provider.py
--
  | mypy: "types.py" shadows library module "types"
  | note: A user-defined top-level module with name "types" is not supported
  | 🚨 Error: The command exited with status 2
```